core/chaincode/shim/crypto/ac/abac.go

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
@christo4ferris your right we will work adding more tests to this functionality. About the header I have modified this file in a recent commit.